specification/manufacturingdatasolution/mds-service/data-plane/Microsoft.Industries.ManufacturingDataSolution/stable/2025-03-01/openapi.json

```diff
@@ -0,0 +1,3593 @@
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/admin/version/{versionId}": {
+      "delete": {
+        "operationId": "DataVersionApis_DeleteVersion",
+        "description": "Delete a data version.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "versionId",
+            "in": "path",
+            "description": "The identifier for the data version.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The request has been accepted for processing, but processing has not yet completed.",
+            "schema": {
+              "type": "object",
+              "description": "Provides status details for long running operations.",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "The unique ID of the operation."
+                },
+                "status": {
+                  "$ref": "#/definitions/Azure.Core.Foundations.OperationState",
+                  "description": "The status of the operation"
+                },
+                "error": {
+                  "$ref": "#/definitions/Azure.Core.Foundations.Error",
+                  "description": "Error object that describes the error when status is \"Failed\"."
+                }
+              },
+              "required": [
+                "id",
+                "status"
+              ]
+            },
+            "headers": {
+              "Operation-Location": {
+                "type": "string",
+                "format": "uri",
+                "description": "The location for monitoring the operation state."
+              },
+              "background-jobid": {
+                "type": "string",
+                "description": "The background job identifier."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataVersionApis_DeleteVersion": {
+            "$ref": "./examples/DataVersionApis_DeleteVersion.json"
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "operation-location"
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/admin/version:switch": {
+      "post": {
+        "operationId": "DataVersionApis_SwitchVersion",
+        "description": "Switch the current data version.",
```

Comment:
I don't understand what this operation does from this description. Switch the current data version to what?